lib/std/Target.zig

```diff
@@ -1033,10 +1035,10 @@ pub const ObjectFormat = enum {
     // LLVM tags deliberately omitted:
     // - dxcontainer
 
-    pub fn fileExt(of: ObjectFormat, arch: Cpu.Arch) [:0]const u8 {
+    pub fn fileExt(of: ObjectFormat, abi: Abi, arch: Cpu.Arch) [:0]const u8 {
```

Comment:
Given how much `Target` data this function now needs, I think it's time to consider adding an `objFileExt()` helper function on `Target`, similar to what's done for the other functions of this kind.